Tests/UnitTests/Purchasing/Purchases/PurchasesConfiguringTests.swift

```diff
@@ -362,6 +362,91 @@ class PurchasesConfiguringTests: BasePurchasesTests {
         expect(self.paymentQueueWrapper.delegate) === self.purchasesOrchestrator
     }
 
+    func testCustomEntitlementComputationSkipsFirstDelegateCall() throws {
+        self.systemInfo = MockSystemInfo(finishTransactions: true,
+                                         customEntitlementsComputation: true)
+        self.setupPurchases()
+
+        expect(self.purchasesDelegate.customerInfoReceivedCount).toEventually(equal(0))
+    }
+
+    func testWithoutCustomEntitlementComputationDoesntSkipFirstDelegateCall() throws {
+        self.systemInfo = MockSystemInfo(finishTransactions: true,
+                                         customEntitlementsComputation: false)
+        self.setupPurchases()
+
+        expect(self.purchasesDelegate.customerInfoReceivedCount).toEventually(equal(1))
+    }
+
+    func testConfigureWithCustomEntitlementComputationFatalErrorIfNoAppUserID() throws {
+        let expectedMessage = Strings.configure.custom_entitlements_computation_enabled_but_no_app_user_id.description
+
+        expectFatalError(expectedMessage: expectedMessage, testcase: {
+            Purchases.configure(
+                with: .builder(withAPIKey: "apiKey")
+                    .with(dangerousSettings: DangerousSettings(customEntitlementComputation: true))
+                    .build())
+        })
```

Comment:
I'm puzzled about this one, will dig in again tomorrow. But basically this one is failing because the SDK doesn't consider this anonymous... And it doesn't consider it anonymous because it sees an appUserID with value `app_user_id`, which must be leaked from another test

Reply:
same result if I use `setupAnonPurchases` instead

Reply:
nevermind, I'll just move the check